src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -56,6 +56,11 @@ public void Loaded(ViewLoadedParams viewLoadedParams)
         {
             if (viewLoadedParams == null) throw new ArgumentNullException(nameof(viewLoadedParams));
 
+            this.ViewModel.MessageLogged += (msg) =>
```

Comment:
can we unsubscribe this handler in the dispose method?

Reply:
Sure. I'll do that.